test/fixtures/parser/ansi/like_operators.yml

```diff
@@ -1,38 +1,38 @@
-file:

```

Comment:
I keep seeing these large diffs. I assume it's because of platform-specific line endings. If so, think we should change the `test/generate_parse_fixture_yml.py` script to use consistent (probably Unix) line endings regardless of where it runs?

Reply:
I went ahead and had a go at this. See #988.